src/handlers/interventions-processor-handler.ts

```diff
@@ -180,11 +200,24 @@ async function validateAccountIsNotDeleted(
   if (itemFromDB?.isAccountDeleted === true) {
```

Comment:
```suggestion
  if (itemFromDB?.isAccountDeleted) {
```

Reply:
Hi @KG700 , @amd-gds - leave as-is just to make it explicit otherwise it be misinterpreted AND to guard against any non-falsy values inadvertently being interpreted as true.